client/src/app/components/login-form/login-form.component.scss

```diff
@@ -28,9 +38,19 @@
     text-align: center;
     max-width: none;
 
+    ::ng-deep .mat-form-field-appearance-outline .mat-form-field-infix {
+      height: 35px;
+    }
```

Comment:
I wanted to make changes to mobile version specifically to reduce button heights so that we can have enough space to show error messages in small devices like iphone 5. But to be consistent with desktop version I will change it in the future commit.